docs/swagger.yaml

```diff
@@ -378,6 +376,13 @@ components:
           - image/svg+xml
           - text/targetUrl
       description: 'Requested image format. Supported values: {image/*,image/png, image/svg+xml, text/targetUrl}'
+    Size:
+      name: Size
+      in: header
+      required: false
+      schema:
+        type: string
```

Comment:
could add a "pattern" to only allow numbers between 100-2000

Reply:
Regex pattern on a string to check a number? Hehe. 
I think your idea from earlier today is better, we should try to set it as a integer.
`    Size:
      name: Size
      in: header
      required: false
      schema:
        type: integer
        minimum: 100
        maximum: 2000
`
Maybe this will work?

Reply:
It will only break for merchants that send in the current header wrongly, right? If somebody sends 500 now, it should continue working?

Reply:
yes i believe so. But right now as far as I can see, I see no restrictions on the size. so somebody sending 99 will not work anymore if we put the minimum and maximum values

Reply:
And test in uat that it still works without the header 😅